features/polyExplorer/src/components/dataViz/purposesBarChart.jsx

```diff
@@ -145,18 +147,18 @@ const PurposesBarChart = ({ data, animation, onClick }) => {
                 .enter()
                 .append("foreignObject")
                 .attr("x", labelXPosition)
-                .attr("y", (d) => yScale(d.title) + gHeight)
+                .attr("y", (d) => yScale(d.title) + labelTitleYAdjust)
                 .attr("width", width)
-                .attr("height", labelTitleHeight.fo)
+                .attr("height", 36)
```

Comment:
Is this related to margin.top? Is there any other constant that could be used here?

Reply:
No, it's still `labelTitleHeight.fo`. I forgot to change the value where it should be, thanks for noticing! :)